<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- reset the default iGEM look -->
    <link rel="stylesheet" href="css/igem_reset.css">

    <!-- materialize -->
    <link rel="stylesheet" href="css/materialize.css">
    <script src="js/materialize.js"></script>

    <!-- Animate on scroll -->
    <link rel="stylesheet" href="css/aos.css">
    <script src="js/aos.js"></script>

    <!-- MathJax -->
    <script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

    <!-- stylesheet -->
    <link rel="stylesheet" href="css/style.css">

    <title>Design | IISc Bangalore</title>
</head>

<body>
    <!-- navbar start -->
    <div class="navbar-fixed hide-on-med-and-down">
        <nav>
            <div class="nav-wrapper">
                <ul class="left nav-list valign-wrapper">
                    <li><a id="nav-logo" class="nav-link" href="index.html"><img class="responsive-img icon"
                                src="assets/images/team_iisc_logo_min.png" style="max-height: 60px;"></a></li>

                    <li><a class="nav-link dropdown-trigger" data-target="dropdown-project" href="#">Project</a></li>

                    <li><a class="nav-link nav-tab" href="Parts.html">Parts</a></li>

                    <li><a class="nav-link dropdown-trigger" data-target="dropdown-labwork" href="#">Wetlab</a></li>

                    <li><a class="nav-link dropdown-trigger" data-target="dropdown-drylab">Drylab</a></li>

                    <li><a class="nav-link dropdown-trigger" data-target="dropdown-hp">Human Practices</a></li>

                    <li><a class="nav-link dropdown-trigger" data-target="dropdown-team" href="#">Team</a></li>

                    <li><a class="nav-link nav-tab" href="Judging.html">Judging</a></li>
                </ul>

                <ul id="dropdown-project" class="dropdown-content">
                    <li class="dropdown-item"><a class="dropdown-link" href="Description.html">Description</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Design.html">Design</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Engineering.html">Engineering</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Implementation.html">Implementation</a>
                    </li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Proof_Of_Concept.html">Proof of Concept</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="UN_SDG.html">UN SDG</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Safety.html">Safety</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Contribution.html">Contribution</a></li>
                </ul>

                <ul id="dropdown-labwork" class="dropdown-content">
                    <li class="dropdown-item"><a class="dropdown-link" href="Results.html">Results</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Notebook.html">Lab Notebook</a></li>
                </ul>

                <ul id="dropdown-drylab" class="dropdown-content">
                    <li class="dropdown-item"><a class="dropdown-link" href="Model.html">Model</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Predictions.html">Protein Structure
                            Predictions</a></li>
                </ul>

                <ul id="dropdown-hp" class="dropdown-content">
                    <li class="dropdown-item"><a class="dropdown-link" href="Communication.html">Public Education and
                            Engagement</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Human_Practices.html">Integrated Human
                            Practices</a></li>
                </ul>

                <ul id="dropdown-team" class="dropdown-content">
                    <li class="dropdown-item"><a class="dropdown-link" href="Members.html">Members</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Attributions.html">Attributions</a></li>
                    <li class="dropdown-item"><a class="dropdown-link" href="Collaborations.html">Collaborations</a>
                    </li>
                </ul>
            </div>
        </nav>
    </div>

    <div id="loader"></div>

    <div class="header hide-on-med-and-down">
        <div class="progress-container">
            <div class="progress-bar" id="myBar"></div>
        </div>
    </div>

    <div class="nav-fixed hide-on-large-only">
        <div class="accordian">
            <button class="accordion">Navigation dropdown</button>
            <div class="panel">
                <ol>
                    <li><a class="mobile-nav-link" href="index.html">Home</a></li>
                    <li>
                        <a class="mobile-nav-link">Project:</a>
                        <ol style="list-style-type: lower-latin;">
                            <li><a class="mobile-nav-link" href="Description.html">Description</a></li>
                            <li><a class="mobile-nav-link" href="Design.html">Design</a></li>
                            <li><a class="mobile-nav-link" href="Engineering.html">Engineering</a></li>
                            <li><a class="mobile-nav-link" href="Implementation.html">Implementation</a></li>
                            <li><a class="mobile-nav-link" href="Proof_Of_Concept.html">Proof of Concept</a></li>
                            <li><a class="mobile-nav-link" href="UN_SDG.html">UN SDG</a></li>
                            <li><a class="mobile-nav-link" href="Safety.html">Safety</a></li>
                            <li><a class="mobile-nav-link" href="Contribution.html">Contribution</a></li>
                        </ol>
                    </li>
                    <li><a class="mobile-nav-link" href="Parts.html">Parts</a></li>
                    <li>
                        <a class="mobile-nav-link">Lab work:</a>
                        <ol style="list-style-type: lower-latin;">
                            <li><a class="mobile-nav-link" href="Results.html">Results</a></li>
                            <li><a class="mobile-nav-link" href="Notebook.html">Lab notebook</a></li>
                        </ol>
                    </li>
                    <li>
                        <a class="mobile-nav-link" href="#">Drylab</a>
                        <ol style="list-style-type: lower-latin;">
                            <li><a class="mobile-nav-link" href="Model.html">Model</a></li>
                            <li><a class="mobile-nav-link" href="Predictions.html">Protein Structure Predictions</a>
                            </li>
                        </ol>
                    </li>
                    <li>
                        <a class="mobile-nav-link">Human Practices:</a>
                        <ol style="list-style-type: lower-latin;">
                            <li><a class="mobile-nav-link" class="nav-link" href="Communication.html">Public Education
                                    and Engagement</a></li>
                            <li><a class="mobile-nav-link" class="nav-link" href="Human_Practices.html">Integrated Human
                                    Practices</a></li>
                        </ol>
                    </li>
                    <li>
                        <a class="mobile-nav-link">Team:</a>
                        <ol style="list-style-type: lower-latin;">
                            <li><a class="mobile-nav-link" href="Members.html">Members</a></li>
                            <li><a class="mobile-nav-link" href="Attributions.html">Attributions</a></li>
                            <li><a class="mobile-nav-link" href="Collaborations.html">Collaborations</a></li>
                        </ol>
                    </li>
                    <li><a class="mobile-nav-link" href="Judging.html">Judging</a></li>
                </ol>
            </div>
        </div>
    </div>

    <div class="hide-on-med-and-down">
        <button onclick="topFunction()" id="myBtn" title="Go to top">Back to Top</button>
    </div>
    <!-- navbar end -->

    <style>
        .hero-image {
            background-image: url("assets/backgrounds/design.jpg");
            height: 750px !important;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
        }

        .hero-text {
            text-align: center;
            position: absolute;
            top: 50%;
            left: 50%;
            transform: translate(-50%, -50%);
            color: white !important;
        }
    </style>

    <div class="hero-image" data-aos="zoom-in">
        <div class="hero-text">
            <h1 style="color: white">Design</h1>
        </div>
    </div>
    <br>

    <div class="container">
        <div class="row">
            <div class="col s12 abstract-box" data-aos="fade-right">
                This section describes the design of the main components conforming CellOPHane, including overview of
                technical details that guided us when we were designing various aspects of this project.
            </div>
            <div class="col s12 l6">
                <div class="card" style="background: rgb(248, 249, 250); border-radius: 2px;" data-aos="fade-right">
                    <div class="card-content">
                        <span class="card-title" style="font-weight: bold; font-size: xx-large;">Contents:</span>
                        <ul>
                            <li><a href="#protein" class="content-link">Proteins</a></li>
                            <li><a href="#cellulose" class="content-link">Bacterial Cellulose Sheet</a></li>
                            <li><a href="#func" class="content-link">Functionalization</a></li>
                            <li><a href="#plasmid" class="content-link">Plasmid</a></li>
                            <li><a href="#chassis" class="content-link">Chassis</a></li>
                            <li><a href="#codon" class="content-link">Codon Adaptation</a></li>
                            <li><a href="#modular" class="content-link">Modular Filtration Platform</a></li>
                            <li><a href="#gibson" class="content-link">Gibson Assembly</a></li>
                            <li><a href="#hardware" class="content-link">Hardware</a></li>
                            <li><a href="#ref" class="content-link">References</a></li>
                        </ul>
                    </div>
                </div>
            </div>

            <div class="col s12" data-aos="fade-right">
                <p>
                    The very first question every iGEM team needs to answer is what their project should be about. Our
                    goal was not only to participate successfully in the iGEM competition but to give something back to
                    both the synthetic biology community and our local community. Organophosphate (OP) pollution is a
                    major issue not only in India but also all over the globe. This inspired us to focus on the
                    degradation of organophosphates by implementation of synthetic biology. Our approach does not
                    suggest that OP pollution should continue, but it offers an environmentally friendly way to fight
                    the existing OP-polluted water.
                </p>
            </div>
            <div id="protein" class="col s12" data-aos="fade-right">
                <h2>Proteins: <br> Our Workhorses</h2>
                <p>
                    The proteins with which we intend to functionalize the bacterial cellulose (BC) filter should meet
                    three functional aspects:
                </p>
                <ol>
                    <li>
                        <p>
                            It must attach to the BC substratum strongly enough that it does not get washed away with
                            the flow of water
                        </p>
                    </li>
                    <li>
                        <p>
                            It must catalytically degrade harmful organophosphate molecules into relatively harmless
                            chemicals which can be easily separated from the water sample (say, with help of some
                            physical process).
                        </p>
                    </li>
                    <li>
                        <p>
                            It must be modular so that other teams in future can design some other bioremediation
                            process (for example, addressing organochlorine pollution) simply by doing a minor change to
                            our filtering platform.
                        </p>
                    </li>
                </ol>
                <p>
                    These criteria were enough to convince us on designing fusion proteins in order to achieve our goal.
                    Our endeavours yielded blueprints of two fusion proteins, namely C1 and C2.
                </p>
                <ol>
                    <li>
                        <p>
                            The fusion protein that C1 codes for is essentially the docking module.
                        </p>
                        <img class="responsive-img" src="assets/images/design/Map_C1.png">
                        <p>
                            The N-terminal has a well-characterized <em>pelB</em> leader sequence (19 amino acid long)
                            which
                            helps the protein to get exported to the periplasmic space of Gram-negative bacteria (often
                            used in genetic engineering) via Sec translocation pathway. After aiding in secretion, the
                            signal is removed by a signal peptidase present in the periplasm.
                        </p>
                        <p>
                            The signal peptide is followed by 10x-Histidine (10x-His) tag which can later be used to
                            extract (as well as purify) the protein using lighter transition metal-based affinity
                            columns. The 10x-His tag was chosen instead of the traditional 6x-His tag as the former
                            conveys increased binding affinity for Ni resin and higher extent of purification compared
                            to the latter
                        </p>
                        <p>
                            The 10x-His tag is followed by SpyCatcher002. SpyCatcher002 is separated from 10x-His tag by
                            the GS Linker 1 so that these two do not interfere with each other's function.

                            SpyCatcher002 is followed by dCBD. These two are separated by GS Linker 2 (reason same as
                            previous instance). dCBD or double cellulose binding domain is a fusion of 2 different CBDs
                            of cellobiohydrolases found in <em>Trichoderma reesei</em>. It was chosen as its binding to
                            microcrystalline cellulose is synergistic. Thus, its binding efficiency to cellulose is much
                            higher than single (typical) CBDs. Also, binding is relatively unaltered by changes in time
                            and temperature. The two CBDs are denoted as CBHII CBD and CBHI CBD. dCBD comes with linker
                            between CBHI CBD and CBHII CBD, so we did not have to add any extra linker sequence with it.
                        </p>
                    </li>
                    <li>
                        <p>
                            The fusion protein that C2 codes for is essentially the functional module.
                        </p>
                        <img class="responsive-img" src="assets/images/design/Map_C2.png">
                        <p>
                            The N-terminal features the organophosphate hydrolase OpdA along with its signal peptide
                            (i.e. <em>opdA</em> leader sequence). OpdA is a phosphotriesterase from <em>Agrobacterium
                                radiobacter</em>
                            P230 with broad substrate range (e.g. coumaphos, coroxon, parathion, diazinon, dMUP etc.).
                            We noticed in literature that the specificity constant of OpdA is much higher than that of
                            OPH (a well-studied and similar enzyme from <em>Pseudomonas diminuta</em>), in context of
                            parathion-methyl degradation. As mentioned in the "Background" page, parathion-methyl was
                            the second most consumed OP insecticide used over the 2005-2010 period in India and it
                            belongs to the most dreaded Class Ia! Thus, OpdA was chosen instead of the more popular
                            counterpart OPH.
                        </p>
                        <p>
                            SpyTag002 follows OpdA, followed by a 22 amino acid-long linker. Instead of designing
                            OpdA-SpyCatcher002 fusion and SpyTag002-dCBD fusion, we have gone for OpdA- SpyTag002 and
                            SpyCatcher002-dCBD. SpyTag002 is much shorter than SpyCatcher002, and thus the former is
                            less likely to interfere with the activity of OpdA than the latter.
                        </p>
                        <p>
                            SpyTag002 is followed by a short GS linker, and then a superfolder green fluorescent protein
                            (i.e. sfGFP). It is a basic (i.e. constitutively fluorescent) GFP derived from <em>Aequorea
                                victoria</em> with excitation wavelength of 485 nm, and emission wavelength of 510 nm.
                            It was
                            added to C2 construct as it would facilitate the quantification of fusion protein expression
                            and would help detecting the degree of success of post-hoc functionalization process.
                        </p>
                        <p>
                            This sfGFP is followed by another short GS linker and then 10x-His tag (same purpose as
                            mentioned previously).
                        </p>
                    </li>
                </ol>
            </div>

            <div id="cellulose" class="col s12" data-aos="fade-right">
                <h2>Bacterial Cellulose Sheet: <br> Base for Our Filter</h2>
                <p>
                    Bacterial cellulose (BC) is the ideal eco-friendly, biodegradable, durable base for the filter we
                    seek to build. It is chemically pure, containing no hemicellulose, pectin, wax, or lignin. In spite
                    of possessing high tensile strength, it is flexible. It is more crystalline compared to
                    plant-derived
                </p>
                <p>
                    cellulose, so usage of dCBD yields more benefit when used in conjunction with BC. Also, BC
                    production is not directly dependent on trees. The cellulose microfibrils of BC are significantly
                    smaller than those of plant cellulose, making BC much more porous. Thus, post-hoc functionalization
                    is more effective for BC than plant cellulose. BC-producing bacteria produce cellulose as pellicles
                    which, when still wet, can easily be disintegrated into loose nanofibrils. However, one cannot
                    produce nanofibrils from phytocellulose without a cumbersome homogenization process.
                </p>
                <p>
                    We plan to utilize BC-producing bacteria to obtain BC pellicles. These can be mechanically
                    disintegrated into loose nanofibrils. The randomly aligned BC nanofibrils can be easily oriented to
                    get a sheet if a stress is applied to those while drying.
                </p>
            </div>

            <div id="func" class="col s12" data-aos="fade-right">
                <h2>Functionalization: <br> Our Road to Activating the Filter</h2>
                <p>
                    Since the chemical composition of BC is similar to phytocellulose, it can also be carboxymethylated,
                    acetylated, phosphorylated, and modified by other graft copolymerization and crosslinking reaction
                    to obtain a series of BC derivatives. The introduction of new functional groups to the BC structure
                    can endow BC with superior mechanical properties or/and stability against ready microbial
                    degradation, while maintaining the unique three-dimensional nano network and porous structure.
                </p>
                <p>
                    Then, we plan to incubate the BC sheet and the fusion proteins (i.e. both C1 and C2) together in
                    order to make the OP-degrading filter material.
                </p>
            </div>

            <div id="plasmid" class="col s12" data-aos="fade-right">
                <h2>Plasmid</h2>
                <p>
                    We have used pETMCN-T7 (KanR) vector, because it has a high copy number and transcription using the
                    T7 RNA polymerase can be strongly induced upon addition of IPTG (i.e. isopropyl
                    \(\beta\)-D-1-thiogalactopyranoside). Also, the pET vector system is a powerful and widely used system for
                    expressing recombinant proteins in <em><em>E. coli</em></em>, a well-characterized and reliable chassis. A
                    powerful
                    feature of the pET system is the ability to clone target gene under conditions of extremely low
                    transcriptional activity, that is, in the absence of a source of T7 RNA polymerase. Background
                    expression is minimal in the absence of T7 RNA polymerase because the host RNA polymerases do not
                    initiate from T7 promoters and the cloning sites in pET plasmids are in regions weakly transcribed
                    (if at all) by read-through activity of bacterial RNA polymerase.
                </p>
            </div>

            <div id="chassis" class="col s12" data-aos="fade-right">
                <h2>Chassis</h2>
                <p>
                    Target genes are initially cloned into the vector using hosts that do not contain the T7 RNA
                    polymerase gene, thus eliminating plasmid instability due to the production of proteins potentially
                    toxic to the host cell. We have chosen <em><em>E. coli</em></em> DH5\(\alpha\) as the host for cloning and
                    plasmid
                    amplification. It lacks T7 RNA polymerase gene, is <em>recA<sup>-</sup></em>
                    <em>endA<sup>-</sup></em>, and gives high transformation
                    efficiencies as well as good plasmid yields.
                </p>
                <p>
                    We have chosen <em><em>E. coli</em></em> BL21(DE3) as the host for expression. This strain is a lysogen of
                    bacteriophage DE3, a lambda derivative that has the immunity region of phage <em>21</em> and
                    carries a DNA
                    fragment containing the <em>lacI</em> gene, the <em>lacUV5</em> promoter (inducible by IPTG), and a
                    chromosomal copy
                    of the gene for T7 RNA polymerase. As an <em><em>E. coli</em></em> B strain, BL21 is deficient in the
                    <em>lon</em> protease and
                    lacks the OmpT outer membrane protease that can degrade proteins during purification.
                </p>
            </div>

            <div id="codon" class="col s12" data-aos="fade-right">
                <h2>Codon Adaptation</h2>
                <p>
                    Some of the proteins coded by the sequences designed by us are native to organisms other than E.
                    coli, our chassis. Thus, we have codon optimized the nucleotide sequences (e.g. SpyTag002,
                    SpyCatcher002, OpdA, dCBD, linkers) for <em>E. coli</em> by using the 'Choose Alternative Codons' capability
                    and codon usage tables available in SnapGene while designing the constructs.
                </p>
            </div>

            <div id="modular" class="col s12" data-aos="fade-right">
                <h2>Modular Filtration Platform: <br> Our Contribution to Bioremediation Strategies</h2>
                <p>
                    As discussed earlier, modularity is our key to building a plug-n-play bioremediation platform. It is
                    an indispensable aspect of synthetic biology as well. We have incorporated the SpyTag/SpyCatcher
                    technology, also termed "molecular superglue", in our design of the fusion proteins (i.e. C1 and C2)
                    in order to ensure modularity. This is a technology for irreversible conjugation of recombinant
                    proteins. The peptide SpyTag spontaneously reacts, both autocatalytically and quantitatively, with
                    the protein SpyCatcher to form an intermolecular isopeptide bond (a covalent bond) between the pair.
                    We have used SpyTag002/SpyCatcher002 instead of the original SpyTag001/SpyCatcher001, as the former
                    is a second-generation, faster-reacting version of this protein superglue at cellular expression
                    levels. We have not used the latest SpyTag003/SpyCatcher003 system because it already exists in the
                    iGEM Registry of Standard Biological Parts. Thus, we intend to submit this codon optimized
                    SpyTag002/SpyCatcher002 system in the iGEM Registry as a contribution from our side to the synthetic
                    biology community.
                </p>
                <p>
                    Proteins to bind or degrade a limitless range of contaminants can be bound to cellulose membranes.
                    In that sense, our filtration platform is inherently modular, thanks to SpyTag002/SpyCatcher002
                    system. If some future group of workers intend to construct a filter to address some pollution other
                    than organophosphate pollution, they would have to replace the OpdA part of C2 with their enzyme of
                    interest. This modified construct would still be compatible with C1 designed by us. That modified C2
                    and original C1 would together constitute a system to neutralize that other pollution. We
                    specifically encourage the future iGEM teams interested in bioremediation to adopt this strategy and
                    to add to this novel plug-n-play toolbox through their invaluable efforts as well as boundless
                    creativity.
                </p>
            </div>

            <div id="gibson" class="col s12" data-aos="fade-right">
                <h2>Gibson Assembly: <br> Our Working Tool</h2>
                <p>
                    Gibson assembly allows for successful assembly of multiple DNA fragments in a single-tube reaction,
                    regardless of fr

                    for large DNA constructs. It enables directional cloning, is very rapid and efficient. Most
                    importantly, it is a seamless method and does not yield any scars. Scars are detrimental to the
                    proper functioning of fusion proteins, the very foundation of CellOPHane. Thus, we have used this
                    elegant cloning technique.
                </p>
            </div>

            <div id="hardware" class="col s12" data-aos="fade-right">
                <h2>Hardware: <br> Our Ultimate Goal</h2>
                <p>
                    Down the road, we seek to build a functional, cost-effective, reliable filtration device which can
                    be installed in the regions where waterbodies are plagued with organophosphate pollution. The basic
                    idea is to coat the inner surface (bore) of microchannels with the functionalized filter material we
                    plan to manufacture, and then to pass the water polluted with OP through an array of these coated
                    microchannels. We depend on principles of hydrodynamics and microfluidics to come up with
                    microchannels of specific shape(s) that ensure adequate residence time of OP molecules inside the
                    channel, so that the OpdA molecules get sufficient time to act upon those.
                </p>
                <p>
                    Once we are ready with the prototype of our Open-source Hardware (OSH), we intend to proceed for its
                    licensing and release into the SynBio community.
                </p>
            </div>

            <div id="ref" class="col s12" data-aos="fade-right">
                <h2>References</h2>
                <ol>
                    <li>Sletta H, Tondervik A, Hakvag S, Vee Aune TE, Nedal A, Aune R, Evensen G, Valla S, Ellingsen TE,
                    Brautaset T. The Presence of N-Terminal Secretion Signal Sequences Leads to Strong Stimulation of
                    the Total Expression Levels of Three Tested Medically Important Proteins during High-Cell-Density
                    Cultivations of <em>Escherichia coli</em>. <em>Appl. Environ. Microbiol</em>. 2007;73(3):906-912.
                    doi:10.1128/AEM.01804-06</li>

                    <li>Zhang W, Lu J, Zhang S, Liu L, Pang X, Lv J. Development an effective system to expression
                    recombinant protein in <em>E. coli</em> via comparison and optimization of signal peptides: Expression of
                    <em>Pseudomonas fluorescens</em> BJ-10 thermostable lipase as case study. <em>Microb. Cell Fact</em>. 2018 Mar
                    28;17(1):50. doi: 10.1186/s12934-018-0894-y</li>

                    <li>Lee J, Kim SH. High-throughput T7 LIC Vector for Introducing C-terminal Poly-Histidine Tags with
                    Variable Lengths without Extra Sequences. <em>Protein Expr. Purif</em>. 2009;63(1):58-61.
                    doi:10.1016/j.pep.2008.09.005</li>

                    <li>Zakeri B, Fierer JO, Celik E, Chittock EC, Schwartz-Linek U, Moy VT, Howarth M. Peptide Tag
                    Forming a Rapid Covalent Bond to a Protein, through Engineering a Bacterial Adhesin. <em>Proc. Natl.
                        Acad. Sci</em>. U.S.A. 2012;109(12):E690-7. doi: 10.1073/pnas.1115485109</li>

                    <li>Keeble AH, Banerjee A, Ferla MP, Reddington SC, Anuar INAK, Howarth M. Evolving Accelerated
                    Amidation by SpyTag/SpyCatcher to Analyze Membrane Dynamics. Angew. <em>Chem. Int. Ed. Engl</em>. 2017 Dec
                    22;56(52):16521-16525. doi: 10.1002/anie.201707623</li>

                    <li>Linder M, Salovuori I, Ruohonen L, Teeri TT. Characterization of a Double Cellulose-Binding
                    Domain: Synergistic High Affinity Binding to Crystalline Cellulose. <em>J. Biol. Chem</em>. 1996 Aug
                    30;271(35):21268-72. doi: 10.1074/jbc.271.35.21268</li>

                    <li>Horne I, Sutherland TD, Harcourt RL, Russell RJ, Oakeshott JG. Identification of an opd
                    (organophosphate degradation) Gene in an Agrobacterium Isolate. <em>Appl. Environ. Microbiol</em>. 2002 Jul
                    01;68(7):3371-3376. doi: 10.1128/aem.68.7.3371-3376.2002</li>

                    <li>Dumas DP, Durst HD, Landis WG, Raushel FM, Wild JR. Inactivation of Organophosphorus Nerve Agents
                    by the Phosphotriesterase from <em>Pseudomonas diminuta</em>. <em>Archives of Biochemistry and Biophysics</em>. 1990
                    Feb 15;277(1):155-159. doi: 10.1016/0003-9861(90)90564-F</li>

                    <li>Pedelacq JD, Cabantous S, Tran T, Terwilliger TC, Waldo GS. Engineering and Characterization of a
                    Superfolder Green Fluorescent Protein. <em>Nat. Biotechnol</em>. 2006 Jan;24(1):79-88. doi: 10.1038/nbt1172</li>

                    <li>Gorgieva S, Trcek J. Bacterial Cellulose: Production, Modification and Perspectives in
                    Biomedical Applications. <em>Nanomaterials</em>. 2019 Sep 20;9(1352). doi:10.3390/nano9101352</li>

                    <li>Lee KY, Buldum G, Mantalaris A, Bismarck A. More Than Meets the Eye in Bacterial Cellulose:
                    Biosynthesis, Bioprocessing, and Applications in Advanced Fiber Composites. <em>Macromol. Biosci</em>. 2014
                    Jan;14(1):10-32. doi: 10.1002/mabi.201300298</li>

                    <li>Novagen. "pET System Manual" (8th Ed.). TB055; 02/99.</li>

                    <li>Hu W, Chen S, Yang J, Li Z, Wang H. Functionalized Bacterial Cellulose Derivatives and
                    Nanocomposites. <em>Carbohydrate Polymers</em>. 2013 Sep. doi: 10.1016/j.carbpol.2013.09.102</li>

                    <li>New England Biolabs Inc. "Gibson Assembly Master Mix: Instruction Manual". DNA Modifying
                    Enzymes (series).</li>

                    <li>iGEM TU Eindhoven 2015 et. al. "The Cloning Guide: The First Step towards a Successful iGEM
                    Project".</li>

                    <li>iGEM Cambridge-JIC 2015. "Open-source Hardware: An iGEM Guide".</li>
                </ol>
            </div>

        </div>
    </div>

    <!-- footer start -->
    <footer class="page-footer">
        <div class="container">
            <h5 class="sm-title center" data-aos="fade-up">Our Sponsors</h5>
            <br>
            <div class="row footer-content valign-wrapper">
                <div class="col s2 l2" data-aos="zoom-in">
                    <a href="https://iisc.ac.in"><img class="responsive-img sponsor-img"
                            src="assets/sponsors/iisc.png"></a>
                </div>
                <div class="col s3 l3" data-aos="zoom-in">
                    <a href="https://benchling.com"><img class="responsive-img sponsor-img"
                            src="assets/sponsors/benchling.png"></a>
                </div>
                <div class="col s3 l3" data-aos="zoom-in">
                    <a href="https://snapgene.com"><img class="responsive-img sponsor-img"
                            src="assets/sponsors/snapgene.png"></a>
                </div>
                <div class="col s2 l2" data-aos="zoom-in">
                    <a href="https://labfolder.com"><img class="responsive-img sponsor-img"
                            src="assets/sponsors/labfolder.png"></a>
                </div>
                <div class="col s2 l2" data-aos="zoom-in">
                    <a href="https://odaa.iisc.ac.in"><img class="responsive-img sponsor-img"
                            src="assets/sponsors/odaa.png"></a>
                </div>
            </div>
            <br>
            <hr>
            <div class="hide-on-med-and-down">
                <div class="row footer-content valign-wrapper">
                    <div class="col s12 l3 center" data-aos="fade-right">
                        <h5 class="sm-title">Our Social Media Links</h5>
                        <a target="_blank" href="https://www.facebook.com/iiscbangaloreigem/"><img
                                src="assets/icons/facebook.png" class="sm-button"></a>
                        <a target="_blank" href="https://twitter.com/igem_iisc?s=08"><img src="assets/icons/twitter.png"
                                class="sm-button"></a>
                        <a target="_blank" href="https://instagram.com/igem_iisc?utm_medium=copy_link"><img
                                src="assets/icons/instagram.png" class="sm-button"></a>
                    </div>
                    <div class="col s12 l3 center" data-aos="zoom-in">
                        <a href="https://2021.igem.org/Team:IISc-Bangalore"><img class="responsive-img footer-logo"
                                src="assets/images/team_iisc_logo_footer.png" style="max-height: 250px"></a>
                    </div>
                    <div class="col s12 l6 center" data-aos="fade-left">
                        <h5 class="sm-title">Contact Us</h5>
                        <p style="text-align: center;">
                            <a href="mailto:igem.ug@iisc.ac.in" class="footer-link">igem.ug@iisc.ac.in</a> or <a
                                href="mailto:sukriyoc@iisc.ac.in" class="footer-link">sukriyoc@iisc.ac.in</a>
                            <br>
                            <br>
                            Office of UG Programme, Indian Institute of Science, <br>
                            C.V. Raman Avenue, Bengaluru - 560 012.
                        </p>
                    </div>
                </div>
            </div>
            <div class="hide-on-large-only">
                <div class="row footer-content">
                    <div class="col s12 l3 center" data-aos="fade-right">
                        <h5 class="sm-title">Our Social Media Links</h5>
                        <a target="_blank" href="https://www.facebook.com/iiscbangaloreigem/"><img
                                src="assets/icons/facebook.png"></a>
                        <a target="_blank" href="https://twitter.com/igem_iisc?s=08"><img
                                src="assets/icons/twitter.png"></a>
                        <a target="_blank" href="https://instagram.com/igem_iisc?utm_medium=copy_link"><img
                                src="assets/icons/instagram.png"></a>
                    </div>
                    <div class="col s12 l3 center" data-aos="zoom-in">
                        <a href="https://2021.igem.org/Team:IISc-Bangalore"><img class="responsive-img"
                                src="assets/images/team_iisc_logo_footer.png" style="max-height: 250px"></a>
                    </div>
                    <div class="col s12 l6 center" data-aos="fade-left">
                        <h5 class="sm-title">Contact Us</h5>
                        <p style="text-align: center;">
                            <a href="mailto:igem.ug@iisc.ac.in" class="footer-link">igem.ug@iisc.ac.in</a> or <a
                                href="mailto:sukriyoc@iisc.ac.in" class="footer-link">sukriyoc@iisc.ac.in</a>
                            <br>
                            <br>
                            Office of UG Programme, Indian Institute of Science, <br>
                            C.V. Raman Avenue, Bengaluru - 560 012.
                        </p>
                    </div>
                </div>
                <div class="row">
                    <h5 style="text-align: center;" class="sm-title">Best viewed on Desktop</h5>
                </div>
            </div>
        </div>
    </footer>

    <script>
        // back to top button
        mybutton = document.getElementById("myBtn");

        window.onload = function () {
            M.AutoInit();
            AOS.init();

            let p_list = document.getElementsByTagName("p")
            // removing top whitespace
            let first_p = p_list[0];
            first_p.style.margin = "0";

            // removing bottom whitespace
            let last_p = p_list[p_list.length - 1];
            last_p.style.margin = "0";

            // removing preloader
            document.getElementById("loader").style.display = "none";
            document.getElementsByTagName("nav")[0].style.display = "block";
            document.getElementsByClassName("header")[0].style.display = "block";

            // preparing accordion
            var acc = document.getElementsByClassName("accordion");
            var i;

            for (i = 0; i < acc.length; i++) {
                acc[i].addEventListener("click", function () {
                    /* Toggle between adding and removing the "active" class,
                    to highlight the button that controls the panel */
                    this.classList.toggle("active");

                    /* Toggle between hiding and showing the active panel */
                    var panel = this.nextElementSibling;
                    if (panel.style.display === "block") {
                        panel.style.display = "none";
                    } else {
                        panel.style.display = "block";
                    }
                });
            }
        }

        window.onscroll = function myFunction() {
            // scroll bar
            var winScroll = document.body.scrollTop || document.documentElement.scrollTop;
            var height = document.documentElement.scrollHeight - document.documentElement.clientHeight;
            var scrolled = (winScroll / height) * 100;
            document.getElementById("myBar").style.width = scrolled + "%";

            // back to top button
            if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
                mybutton.style.display = "block";
            } else {
                mybutton.style.display = "none";
            }
        }

        // back to top button function
        function topFunction() {
            document.body.scrollTop = 0; // For Safari
            document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
        }
    </script>
    <!-- footer end -->
</body>

</html>